DB/mysqli.php

```diff
@@ -498,7 +498,11 @@ function fetchInto($result, &$arr, $fetchmode, $rownum = null)
      */
     function freeResult($result)
     {
-        return is_resource($result) ? mysqli_free_result($result) : false;
```

Comment:
I have had to revert this for wmf to get it to pass their tests . I'm not sure if the problem is on the servers - or only the test env (running hhvm) - https://integration.wikimedia.org/ci/job/wikimedia-fundraising-civicrm/3756/console